src/app/helpers/whitelistApis.js

```diff
@@ -551,6 +551,10 @@ const API_LIST = {
     '/learner/anonymous/otp/v1/generate': {
       checksNeeded: []
     },
+    '/learner/data/v1/form/read': {
```

Comment:
Form/Read should be already existing. This is not the new end-point we are using first time.

Reply:
Not required

Reply:
I checked that if this '/learner/data/v1/form/read' is not added in the  white list APIs, then form API call which is going from csl side to get the supported activity list is not working, giving 403